akka-actor-typed-tests/src/test/scala/akka/actor/typed/coexistence/TypedSupervisingUntypedSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.coexistence
+import akka.actor.Actor
+import akka.actor.testkit.typed.TestException
+import akka.actor.testkit.typed.scaladsl.{ ScalaTestWithActorTestKit, TestProbe }
+import akka.actor.typed.ActorRef
+import akka.actor.typed.scaladsl.Behaviors
+import org.scalatest.{ WordSpec, WordSpecLike }
+import akka.actor.typed.scaladsl.adapter._
+import akka.{ actor => untyped }
+
+import scala.concurrent.duration._
+
+object TypedSupervisingUntypedSpec {
+
+  sealed trait Protocol
+  final case class SpawnUntypedActor(props: untyped.Props, replyTo: ActorRef[SpawnedUntypedActor]) extends Protocol
+  final case class SpawnedUntypedActor(ref: untyped.ActorRef)
```

Comment:
confusing naming, and should this extend `Protocol`?

Reply:
Its the reply, the Protocol is for messages to sent to the actor